config/prod/agora-data-manager-github-runner.yaml

```diff
@@ -0,0 +1,9 @@
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.9.3/templates/ImageBuilder/amazon-linux-2023-agora-bastian.yaml"
+stack_name: "agora-data-manager-github-runner"
```

Comment:
do you think it would be useful to have a dev stack for testing changes, or is it enough to rely on testing the template in aws-infra before creating a new version here?